tests/monitoring/monitoring.go

```diff
@@ -638,8 +638,62 @@ var _ = Describe("[Serial][sig-monitoring]Prometheus Alerts", Serial, func() {
 			tests.WaitForVirtualMachineToDisappearWithTimeout(vmi, 240)
 		})
 	})
+
+	Context("Warning alerts", func() {
+		alerts := []string{
+			"KubeVirtComponentExceedsRequestedCPU",
+			"KubeVirtComponentExceedsRequestedMemory",
+		}
+
+		BeforeEach(func() {
+			virtClient, err = kubecli.GetKubevirtClient()
+			util.PanicOnError(err)
+
+			scales = make(map[string]*autoscalingv1.Scale, 1)
+			backupScale(virtOperator.deploymentName)
+
+			updateScale(virtOperator.deploymentName, 0)
+			reduceAlertPendingTime()
+		})
+
+		AfterEach(func() {
+			revertScale(virtOperator.deploymentName)
+
+			time.Sleep(10 * time.Second)
```

Comment:
This approach looks non-deterministic to me, why not `5` and maybe it can be even `11`? why do you need to wait here? can you use a custom watcher here if needed?

Reply:
removed